platform/ui/src/components/content/downloadContent/DownloadContent.js

```diff
@@ -1,10 +1,9 @@
 import React, { useEffect, useState, createRef } from 'react';
-import Modal from 'react-bootstrap-modal';
 import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
 
-import './DownloadDialog.styl';
+import './DownloadContent.styl';
```

Comment:
Hate to make you rename it again, but this is more like a:

`DownloadViewportForm` or
`ViewportDownloadForm`

Reply:
Done! thx.